frontend/app/src/components/StatusWidget/StatusWidget.tsx

```diff
@@ -136,237 +106,244 @@ const RUNNING_MAN_DISPLAY_DELAY_TIME_MS = 500
  * Displays various script- and connection-related info: our WebSocket
  * connection status, the run-state of our script, and other transient events.
  */
-class StatusWidget extends PureComponent<StatusWidgetProps, State> {
-  /** onSessionEvent signal connection */
-  private sessionEventConn?: SignalConnection
-
-  private curView?: ReactNode
-
-  private readonly minimizePromptTimer = new Timer()
+const StatusWidget: React.FC<StatusWidgetProps> = ({
+  connectionState,
+  sessionEventDispatcher,
+  scriptRunState,
+  rerunScript,
+  stopScript,
+  allowRunOnSave,
+  //theme,
```

Comment:
**suggestion:** Remove this

Reply:
[c6e9678](https://github.com/streamlit/streamlit/pull/9851/commits/c6e9678c785c412e586189b3934b8c92ce023fe4)